src/pages/resultsView/mutation/ResultsViewMutationMapperStore.ts

```diff
@@ -307,6 +336,33 @@ export default class ResultsViewMutationMapperStore extends MutationMapperStore
             }
         });
 
+        // Add categorical namespace column definitions.
+        _.forIn(
+            this.namespaceColumnConfig,
+            (namespaceColumnNames, namespaceName) => {
+                _.forIn(namespaceColumnNames, (type, namespaceColumnName) => {
+                    if (type !== 'number') {
+                        const columnName = createNamespaceColumnName(
+                            namespaceName,
+                            namespaceColumnName
+                        );
+                        columnIds.add(columnName);
+                        this.mutationMapperStoreConfig[
+                            'filterAppliersOverride'
+                        ]![
+                            columnName
+                        ] = createCategoricalFilter((d: Mutation) =>
+                            CategoricalNamespaceColumnFormatter.download(
```

Comment:
maybe a comment explaining why we are using download?   i mean, the problem is perhaps that the "download" is too specific

Reply:
Download here means the value formatted for download. I just copied this from other implementations.